news/10482.bugfix.rst

```diff
@@ -1 +1 @@
-New resolver: Fixes depth ordering of packages during resolution, e.g. a package 2 levels of dependency down will be ordered before a package 3 levels of dependecy down.
+New resolver: Fixes depth ordering of packages during resolution, e.g. a package 2 levels of dependency deep will be ordered before a package 3 levels of dependecy deep.
```

Comment:
```suggestion
New resolver: Fixes depth ordering of packages during resolution, e.g. a dependency 2 levels deep will be ordered before a dependecy 3 levels deep.
```

I feel this makes more sense? (Non-native)

Reply:
I agree.